### DISCUSSION TITLE
Add additional Kartvelian symbols to the Georgian layout

### DISCUSSION THREAD

**Discussion by soshial** [rkkr/simple-keyboard#403] [Ideas]
@rkkr 

There are several linguistic minorities in Georgia (Sakartvelo) that speak their own language related to Georgian. Should I add symbols from their alphabets to the Georgian layout?

Also, the specific script/layout we have for Georgian now has a name called Mkhedruli. Where do I edit it?

**Comment by rkkr** [comment]
The default answer is "look at GBoard".

**Reply by soshial** [reply]
So, if those symbols are not in GBoard, you would reject the changes, right?

**Reply by rkkr** [reply]
Probably yes. I would also expect that GBoard has a separate layout for for such a variant.

**Reply by soshial** [reply]
Unfortunately not, GBoard doesn't support neither Svan, nor Mingrelian, nor Laz language.

**Reply by soshial** [reply]
Example of a separate app with Svan layout: https://play.google.com/store/apps/details?id=com.tevzadze.svanurikeyboard (it's the same layout as Georgian but with mostly vowel diacritics added)
Svan language: https://en.wikipedia.org/wiki/Svan_language
